src/main/java/lt/akademija/itacademymanager/controller/StudentController.java

```diff
@@ -52,12 +56,14 @@ public Student getStudentById(@PathVariable("id") int id) {
         return studentService.getStudentById(id);
```

Comment:
ne visi endpointai apsaugoti, pvz galiu naudotis be prisijungimo getStudentById()